src/lib/style/components/_cards-rotate.scss

```diff
@@ -12,17 +10,8 @@ $card-rotate-margin: 21px;
     & > .SRC-grid-item {
       width: auto;
       height: auto;
-      margin-left: $card-rotate-margin;
-      margin-right: $card-rotate-margin;
       overflow: hidden;
-    }
-
-    & > .SRC-grid-item:first-child {
-      margin-right: $card-rotate-margin;
-    }
-
-    & > .SRC-grid-item:last-child {
-      margin-left: $card-rotate-margin;
+      padding: 5px;
```

Comment:
Another way to force equal width for all 3 columns, maybe use something like `grid-template-columns: repeat(3, minmax(0, 1fr));`?

Reply:
I think we've have the same box-shadow issue if we changed to use your grid definition.

Reply:
The 42px spacing is from the original design, so instead we can apply to the padding instead of the margin withhin each column.

Reply:
Ah, okay. Thanks.